docs/contributors/getting-started.md

```diff
@@ -84,6 +84,34 @@ but most popular are:
 - debugging through logging (recommended option)
 - classic debugging with breakpoints
 
+## Commit messages
+
+Try to follow the
+[conventional commits specification](https://www.conventionalcommits.org/en/v1.0.0/),
+which means your commits should be of form:
+
+```
+<type>[optional scope]: <description>
+
+[optional body]
+
+[optional footer(s)]
+```
+
+Additionally, most should be explained in order for reviewers to understand it
+better or for the future contributors to be able to understand past changes.
```

Comment:
```suggestion
It's important to over-communicate in your commit messages. Be thorough. It's it's important to ensure you
outline the problem you're trying to solve, or the feature your introducing, and then explain the fix or implementation. This will help the reviewers with their review and also future contributors to get the full
context of your changes.
```

What do you think of changing this a bit to something like this?

Reply:
Good idea, applied!